src/main/resources/anet-schema.yml

```diff
@@ -455,6 +455,12 @@ properties:
     title: The maximum number of characters allowed in selected text fields
     description: Used in the UI for report intent, report key outcomes, report next steps, authorization group description
 
+  maxCalendarEventsPerDay:
+    type: integer
+    minimum: 0
+    title: The maximum number of events displayed in a calendar day
+    description: Used in the UI to limit the maximum number of events displayed in the calendar
```

Comment:
From this text it is unclear to me what a minimum of 0 (zero) actually means; no limit? default limit? no rows at all? As stated in the other thread, leaving out the configuration option is fine for now.